Sources/GRPC/AsyncAwaitSupport/GRPCAsyncServerCallContext.swift

```diff
@@ -35,7 +35,7 @@ public final class GRPCAsyncServerCallContext {
   private let lock = Lock()
 
   /// Request headers for this request.
-  public let headers: HPACKHeaders
+  public let requestHeaders: HPACKHeaders
```

Comment:
Concretely, are you suggesting the following?

* `requestMetadata`
* `responseInitialMetadata`
* `responseTrailingMetadata`

Reply:
Yes although I think `initialResponseMetadata` and `trailingResponseMetadata` rather than `responseInitialMetadata` and `responseTrailingMetadata`.

---

Alternatively we could flip the client API to have `resonseHeaders` and `responseTrailers`. I don't mind whether we pick 'metadata' vs 'headers/trailers', the consistency between client and server is more important, I think.

Reply:
[These docs](https://grpc.io/docs/what-is-grpc/core-concepts/) seem to use the "metadata" nomenclature rather than headers and trailers. I'll align with them.

Reply:
Done in https://github.com/grpc/grpc-swift/pull/1262/commits/c5cfcb1bddd7050aae7ce6dbe1f534a08ed2312b